sections/builder.js

```diff
@@ -1526,28 +1526,42 @@ export default [
   {
     type: "method",
     method: "offset",
-    example: ".offset(value)",
-    description: "Adds an offset clause to the query.",
+    example: ".offset(value, options={skipBinding: boolean})",
+    description: "Adds an offset clause to the query. An optional skipBinding parameter may be specified to doesn't add offset as a prepared value (some database doesn't allow prepared value for offset).",
     children: [
       {
         type: "runnable",
         content: `
           knex.select('*').from('users').offset(10)
         `
+      },
+      {
+        type: "runnable",
+        content: `
+          // Offset value isn't a prepared value.
+          knex.select('*').from('users').offset(10, {skipBinding: true}).toSQL().sql
```

Comment:
Can you also add generated SQL for examples both with and without the parameter, so that the difference is clearer?

Reply:
I add them. It's ok to merge this doc ?